lib/game.rb

```diff
@@ -0,0 +1,15 @@
+class Game
+	attr_accessor :board 
+
+	def initialize
+		@board = Array.new(9)
+	end
+
+	def empty_board 
```

Comment:
so empty board is really display board?

Reply:
Yes, the idea here was to allow the user to first view the empty board and then play a turn. But I think it makes sense to use the same method to display the board after every turn/move and rename it to display_board. 
commit 64be04ddd8a7786bd5eda8abfd8fa4256933c192